sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/SendMessageBatchWithConnectionStringSample.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.messaging.servicebus.models.CreateBatchOptions;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class SendMessageBatchWithConnectionStringSample {
+
+    /**
+     * Main method to invoke this demo on how to send an {@link ServiceBusMessageBatch} to an Azure Service Bus.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
```

Comment:
Some of your samples you prefix with Async or SyncSample. It is not clear that it is a sync sample until you delve into the code.

I don't think you need to highlight "ConnectionString" in the title.